internal/controllers/user_integrations_controller.go

```diff
@@ -1511,6 +1511,17 @@ func (udc *UserDevicesController) PostAftermarketDeviceUnpair(c *fiber.Ctx) erro
 		return err
 	}
 
+	// if this is an autopi, mark it as unpaired on their end so they stop billing us
```

Comment:
I don't think anyone calls this anymore. Have you seen any logs?

Reply:
ugh so much old code in devices-api... I just checked and yeah no calls to that in recent history.

I'll deprecate some stuff instead